corehq/apps/locations/forms.py

```diff
@@ -532,9 +533,9 @@ def __init__(self, domain_object, location, *args, **kwargs):
             )
         )
 
-    @property
     @memoized
-    def users_at_location(self):
+    @quickcache(['self.domain_object.name', 'self.location.location_id'], memoize_timeout=0, timeout=5)
```

Comment:
Would it be feasible to make the 5 a constant that's used both here and in ES, or at the bare minimum to add a comment? This seem vulnerable to being undone by some well-meaning future developer.

Reply:
I'm not sure it's necessary that those two be tightly coupled - essentially 5 seconds is the maximum amount of time that these results will be out of date.  We should keep this on a short-ish timeout, since you can assign users to locations in other ways, but it's not a big deal if this exceeds the ES refresh interval.